src/MfaVerifyController.js

```diff
@@ -110,6 +110,11 @@ function (Okta, BaseLoginController, TOTPForm, YubikeyForm, SecurityQuestionForm
         this.listenTo(this.model, 'change:rememberDevice', function (model, rememberDevice) {
           model.get('backupFactor').set('rememberDevice', rememberDevice);
         });
+
+        // Listen to specific errors and bubble them up
+        this.listenTo(this.model, 'errors:verify', function (model, err) {
```

Comment:
Using `errors:verify` as the event name to be consistent with other [Backbone events](http://backbonejs.org/#Events-catalog). Open to changing this internal event if needed.

Reply:
remind me why we cant listen to `this.model # error` event transform as `afterError` ?

Reply:
Good question. We can - however we'd need to validate the return value for a model `error` event in the process. This is just to be more explicit.

For example, to listen to the errors we need:

```javascript
this.listenTo(this.model, 'error', function (model, err) {
  if (!model || !err) { return }

  // We emit an 'error' event for other cases, so we
  // need to check for the error type before emitting
  if (err.name === 'MFA_VERIFY_ERROR') {
    this.trigger(
      'afterError',
      { controller: this.className },
      {
        name: 'MFA_VERIFY_ERROR',
        message: err.responseJSON.errorSummary
      }
    );
  }
});
```

Reply:
hmm, I guess the pro is we don't need to maintain specific error event. what's your thoughts?

Reply:
- we wont need `once` and `bind` at this case, so I'm not sure how that problem is related.
- IMHO, 'event' system add complexity and makes debug harder. and now it's going to have two events when error happened which I thought makes things worse.

Reply:
Pushed another commit which uses the existing `error` event.

It does add some more complexity and requires us to create a new Error type, but I'm assuming that is a good pattern to follow moving forward.

Let me know what you think!